Common/ArmoniK.DevelopmentKit.Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -535,7 +562,22 @@ public byte[] TryGetResult(string taskId, bool checkOutput = true, CancellationT
             msg += $"{string.Join(", ", resultStatus.IdsError)}";
           }
 
-          msg += " ]";
+          msg += $" ]\n";
+
+          var taskIdInError = resultStatus.IdsError.Any() ? resultStatus.IdsError.First() : resultStatus.IdsResultError.First().Item1;
+
+          msg += $"1st task Id {taskIdInError} in error : root cause : \n";
+          var taskStatus = GetTaskStatus(taskIdInError);
+          if (taskStatus is TaskStatus.Error or TaskStatus.Failed)
```

Comment:
TaskStatus.Failed is deprecated and will be removed. It should already never appear.

Reply:
done